package.json

```diff
@@ -121,7 +121,8 @@
       "packages/*"
     ],
     "globals": {
-      "react": "React"
+      "react": "React",
+      "@emotion/core": "emotionCore"
```

Comment:
I don't recall if I added this to whack-a-mole an error or if preconstruct added this automatically, but if this is needed, it probably shouldn't be in root, right?

Probably has something to do with me disabling `@emotion/core`'s UMD.